crates/sui-core/src/authority_active/checkpoint_driver/mod.rs

```diff
@@ -201,6 +201,13 @@ pub async fn checkpoint_process<A>(
             .await;
         }
 
+        if let Err(err) = state_checkpoints
+            .lock()
+            .attempt_to_construct_checkpoint(committee)
+        {
+            warn!("Error attempting to construct checkpoint: {:?}", err);
```

Comment:
Do we proceed by executing the rest of the function if this error happens?

Reply:
I will fix this in a separate PR.